docs/intro.md

```diff
@@ -78,12 +78,13 @@ To run the Lampo daemon with the configuration file located at `~/.lampo/signet/
 
 > **Note**: Before running `lampod`, ensure that Bitcoin Core is already running. Additionally, `lampod` utilizes Bitcoin Core as its wallet.
 
+The first time you run `lampod-cli` with signet network, it will create a new wallet:
```

Comment:
You can do also with `lampod-cli create-wallet` or something similar, look at the `lampod-cli --help`